src/Query.fs

```diff
@@ -9,7 +9,7 @@ let rec readVariableType (variableType: GraphQLType) =
     match variableType.Kind with
     | ASTNodeKind.NamedType ->
         let namedType = unbox<GraphQLNamedType> variableType
-        GraphqlVariableType.Ref namedType.Name.Value
+        GraphqlVariableType.Ref (namedType.Name.Value.ToString())
```

Comment:
Why this `.ToString()` call here?

Reply:
Because they packed the name as ROM struct which is a slice of chars
https://github.com/graphql-dotnet/parser/blob/388da3a1cf98034af73664ecf34ab718783d6044/src/GraphQLParser/ROM.cs#L78